main.js

```diff
@@ -96,11 +90,11 @@ client.on('ready', async () => {
         process.exit(1);
     }
 
-    const markdownText = await getMarkdownText();
+    const markdownText = getMarkdownText();
     const segments = parseMarkdownToSegments(markdownText);
     const embedTexts = segmentsToEmbeds(segments);
-
-    await Promise.all((await channel.fetchMessages({ limit: 100 })).deleteAll());
+    
+    await Promise.all((await channel.fetchMessages({ limit: 20 })).deleteAll());
```

Comment:
Is `20` too few?

Reply:
Current embed count is 9, I think 20 is fine.